checkstyle.xml

```diff
@@ -140,7 +140,6 @@
 
         <!-- Checks for common coding problems               -->
         <!-- See http://checkstyle.sf.net/config_coding.html -->
-        <module name="AvoidInlineConditionals"/>
```

Comment:
Why the change?

Reply:
Because we think that ? operator should be allowed

Reply:
Thank you, I'd agree.